src/sth_server.js

```diff
@@ -560,7 +561,8 @@
                   contextElement = contextResponses[i].contextElement;
                   attributes = contextElement.attributes;
                   for (var j = 0; j < attributes.length; j++) {
-                    if (!attributes[j].value || isNaN(attributes[j].value)) {
+                    if (!attributes[j].value ||
+                      (isNaN(attributes[j].value) && typeof(attributes[j].value) !== 'string')) {
```

Comment:
I understand these conditions are for notified strings that are of type "melones". Nevertheless, does make sense not processing not-number data only because it is not of type string?